project_national/sdr_upgrades_tmy3.yml

```
@@ -1149,4 +1149,28 @@ upgrades:
       - *attic_insulation_IECC_CZ1A
       - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
       - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
-      - *infiltration_30pct_reduction
\ No newline at end of file
+      - *infiltration_30pct_reduction
+
+  - upgrade_name: Upgrade 1 - EV Ownership + L2 Charger
```

Since the upgrade names will be merged with rest of the upgrades best to avoid "Upgrade x" prefix. 
```suggestion
  - upgrade_name: EV Ownership + L2 Charger
```